### PR TITLE
add stress initialization for hydrofracture solver

### PR DESCRIPTION
**Pull request by Guotong-Ren**
prototype the initialization procedure for the hydrofracture solver when stress calculation is needed (e.g., Neumann boundary, nodal forces specified by fieldspecification)

### CODE REVIEW

File: src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp [hunk 1]
```diff
@@ -203,7 +203,7 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::fullyCoupledSolverStep( real
     dtReturn = nonlinearImplicitStep( time_n, dt, cycleNumber, domain );
 
 
-    if( m_surfaceGenerator->solverStep( time_n, dt, cycleNumber, domain ) > 0 )
+    if( time_n >= 0 && m_surfaceGenerator->solverStep( time_n, dt, cycleNumber, domain ) > 0 )
```
Comment (paveltomin) [inline src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp]: this is tricky, i suggest trying to use this flag https://github.com/GEOS-DEV/GEOS/blob/d3783ad15edb16510fed6f4eef214e01ff14ffde/src/coreComponents/physicsSolvers/multiphysics/PoromechanicsSolver.hpp#L232

**Review by paveltomin — APPROVED** [review]
looks fine, hope it works :)

File: src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp (hunk 1 above)
Comment (Guotong-Ren) [reply src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp]: thanks, it is switched to the flag

**Review by jhuang2601 — APPROVED** [review]

**Comment by jhuang2601** [comment]
Try to run `make uncrustify_style` to fix the reported problem for code style

**Comment by Guotong-Ren** [comment]
> Try to run `make uncrustify_style` to fix the reported problem for code style

thanks, Jian, for the suggestion. will do that.

**Pull request merged**